examples/echo.rs

```diff
@@ -43,19 +42,17 @@ fn main() {
     core.run(server).unwrap();
 }
 
-#[async]
-fn handle_client(socket: TcpStream) -> io::Result<u64> {
+#[async_stream(item = u64)]
```

Comment:
Could this stay as it was before? This was mostly intended to showcase the similarity to an otherwise synchronous echo server

Reply:
done